### PR TITLE
[ExtendedLocations] Add tests

### PR DESCRIPTION
**Pull request by HarveyLink**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

File: sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+        [Test]
+        [Ignore("Host resource region does not match Custom Location region")]
+        public async Task TestOperationsCustomLocation()
+        {
+            await SetCollectionsAsync();
+            // Refer to this page to create a hybrid K8s https://docs.microsoft.com/en-us/rest/api/hybridkubernetes/connected-cluster/create?tabs=HTTP
+            string AnsibleTest = "/subscriptions/"+ DefaultSubscription.Data.SubscriptionId + "/resourceGroups/sdktestrg/providers/Microsoft.Kubernetes/connectedClusters/cle2edfkapconnectedcluster/providers/Microsoft.KubernetesConfiguration/extensions/cli-test-operator-ansible";
+            string CassandraTest = "/subscriptions/"+ DefaultSubscription.Data.SubscriptionId + "/resourceGroups/sdktestrg/providers/Microsoft.Kubernetes/connectedClusters/cle2edfkapconnectedcluster/providers/Microsoft.KubernetesConfiguration/extensions/cli-test-operator";
+
+        // CREATE CL
```
Comment (ArthurMa1978) [inline sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs:46]: indent

File: sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        var resourceName = Recording.GenerateAssetName("cltest-");
```
Comment (ArthurMa1978) [inline sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs:47]: indent

**Review by ArthurMa1978 — APPROVED** [review]

File: sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs (hunk 2 above)
Comment (HarveyLink) [reply sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs]: Done

File: sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs (hunk 1 above)
Comment (HarveyLink) [reply sdk/extendedlocation/Azure.ResourceManager.ExtendedLocations/tests/ScenarioTests/CustomLocationOperationTests.cs]: Done

**Pull request merged**